charts/pgadmin4/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: pgAdmin4 is a web based administration tool for PostgreSQL database
 name: pgadmin4
-version: 1.4.1
+version: 1.4.3
```

Comment:
Just wandering, why increase the patch version by 2?

Reply:
Hi,
I double-checked and found my fork repo is even with `rowanruseler:master` branch (commit 9203c60 as latest). Am I missing something?

I increased the version with the assumption that the 2 MRs from qkflies needs to be merged first. At least https://github.com/rowanruseler/helm-charts/pull/62 needs to go in first so that my PR can pass the test workflow. 

Please correct me if I'm wrong. Thank you.

Reply:
I guess CI jobs failure was caused by outdated image `quay.io/helmpack/chart-testing:v2.4.0` used for execution of lint and installation tests. For the commit in PR #61 they were actually not being run, because no changes there detected for the pgadmin chart code. The latest release of [chart-testing](https://github.com/helm/chart-testing) is `3.3.1` at the moment.

@rowanruseler Would you mind to give it a shot by reconfiguring GitHub Actions here?

Reply:
I already adjusted it, but somehow it still fails here. I'll look into it.